string type
string source
string target
string[] arguments
twitch_chat_ros/twitch_tags tags
